src/sorcha/ephemeris/simulation_parsing.py

```diff
@@ -3,6 +3,7 @@
 import numpy as np
 import spiceypy as spice
 
+#from sorcha.ephemeris.simulation_constants import GMSUN, RADIUS_EARTH_KM
```

Comment:
Is this being kept commented out until we determine that it's ok to remove the two lines in `simulation_constants.py`?

Reply:
Yes, but's probably ok to remove it.

Reply:
Removed